packages/common.vm/common.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20250206</version>
+    <version>0.0.0.20250407</version>
```

Comment:
You also need to require this version in the debloat package to ensure the new function is available:
```
<dependency id="common.vm" version="0.0.0.20250407" />
```